### PR TITLE
Add job for compute HA in Cloud 6

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by vuntz** [comment]
Note that it might completely fail; I'll fix things when we have the job running...

File: scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml [hunk 1]
```diff
@@ -0,0 +1,199 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: libvirt
+      libvirt:
+        hypervisor_ip: ""
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml]: Does this make sense to leave empty with libvirt mode? Or are you going to change it to SBD, just like we have in QA jobs?

Comment (vuntz) [inline scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml]: Hrm, no, it's not supposed to be like this (see change I did in qa_crowbarsetup). I guess I lost a change here... Will fix.

Comment (vuntz) [inline scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml]: Force-pushed

**Comment by dirkmueller** [comment]
maybe call it job-ha-compute ?

**Comment by vuntz** [comment]
@dirkmueller yeah. Actually, depending on the output of the discussion (which is not happening yet...) in https://github.com/SUSE-Cloud/automation/pull/835, it would be ha-compute-trigger.

**Comment by dirkmueller** [comment]
@vuntz I personally find the -trigger suffix fairly annoying (and it is no longer needed since they're no longer fire-and-forget jobs but they collec thte test result in the sub job)

**Comment by vuntz** [comment]
> maybe call it job-ha-compute ?

done

**Comment by dirkmueller** [comment]
+1

File: scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+- barclamp: manila
+  attributes:
+    shares:
+    - backend_driver: generic
+      backend_name: backend1
+      generic:
+        service_instance_user: root
+        service_instance_name_or_id: ##manila_instance_name_or_id##
+        service_net_name_or_ip: ##service_net_name_or_ip##
+        tenant_net_name_or_ip: fixed
```
Comment (dirkmueller) [inline scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml]: this needs to be fixed, see https://github.com/SUSE-Cloud/automation/pull/837/files#diff-9e5ac0712fad90190bd522780ee8b7eaR185

Comment (vuntz) [inline scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml]: fixed

**Comment by dirkmueller** [comment]
:+1:

**Comment by aplanas** [comment]
+1

**Pull request merged**